ign_ros2_control/src/ign_ros2_control_plugin.cpp

```diff
@@ -311,19 +311,14 @@ void IgnitionROS2ControlPlugin::Configure(
     }
   }
 
-  std::vector<const char *> argv;
-  for (const auto & arg : arguments) {
-    argv.push_back(reinterpret_cast<const char *>(arg.data()));
-  }
 
+  // Create a default context, if not already
   if (!rclcpp::ok()) {
     rclcpp::init(static_cast<int>(argv.size()), argv.data());
-    std::string node_name = "ignition_ros_control";
-    if (!controllerManagerPrefixNodeName.empty()) {
-      node_name = controllerManagerPrefixNodeName + "_" + node_name;
-    }
-    this->dataPtr->node_ = rclcpp::Node::make_shared(node_name);
   }
+
+  // Instanciate and start node with namespace
+  this->dataPtr->node_ = rclcpp::Node::make_shared("ignition_ros_control", this->dataPtr->robot_namespace_);
```

Comment:
Should the `ignition_ros_control` node name be prefixed the same as controller manager? I'm honestly unsure.

Reply:
@Ronoman This comment made me realized that there are some unnecessary changes in the pull request. I'll try to clean it up a bit to make the PR more readable. As for the comment itself, my understanding is that we are already prefixing both the same way. Here is what I get on create3 and turtlebot4